### PR TITLE
feat(j-s): more multiple loke fixes

### PR DESCRIPTION
**Pull request by Arnarkari93**
[Asana - Mörg LÖKE-númer](https://app.asana.com/0/1199153462262248/1202868512043864/f)

## What

- fix name for request file uploaded to court
- iterate the ready-for-court email notification for the prosecutor
- change the subject of the court date email to the prosecutor
- move the police case number out of the header in the request pdf
-  add a line break between police case numbers in InfoCard

## Why

- Multiple police case numbers iterations

## Checklist:
Screenshot 2022-08-31 at 14 03 17
Screenshot 2022-08-31 at 14 20 43

- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

File: apps/judicial-system/backend/src/app/formatters/requestPdf.ts [hunk 1]
```diff
@@ -89,19 +89,18 @@ function constructRestrictionRequestPdf(
...
-    )} - Mál nr: ${theCase.policeCaseNumbers.join(', ')}`,
-  )
+  addMediumPlusHeading(doc, formatDate(theCase.created, 'PPP') ?? '')
   setLineGap(doc, 40)
   addMediumPlusHeading(
     doc,
     `${formatMessage(m.baseInfo.court)} ${theCase.court?.name}`,
   )
   setLineGap(doc, 4)
+  addMediumText(doc, 'LÖKE málsnúmer', 'Times-Bold')
```
Comment (oddsson) [inline apps/judicial-system/backend/src/app/formatters/requestPdf.ts:99]: Should this be a CF string?

File: apps/judicial-system/backend/src/app/formatters/requestPdf.ts [hunk 2]
```diff
@@ -257,19 +256,18 @@ function constructInvestigationRequestPdf(
...
```
Comment (oddsson) [inline apps/judicial-system/backend/src/app/formatters/requestPdf.ts:266]: Should this be a CF string?

**Review by oddsson — COMMENTED** [review]
Wondering about two strings and if they belong in CF or not

File: apps/judicial-system/backend/src/app/formatters/requestPdf.ts (hunk 1 above)
Comment (Arnarkari93) [reply apps/judicial-system/backend/src/app/formatters/requestPdf.ts]: Good catch, fixed in [057e005](https://github.com/island-is/island.is/pull/8301/commits/057e005ab59c28622522f6883c78dba2dfc4ec5b)

File: apps/judicial-system/backend/src/app/formatters/requestPdf.ts (hunk 2 above)
Comment (Arnarkari93) [reply apps/judicial-system/backend/src/app/formatters/requestPdf.ts]: Fixed in [057e005](https://github.com/island-is/island.is/pull/8301/commits/057e005ab59c28622522f6883c78dba2dfc4ec5b)

**Pull request merged**